test/test-apheleia.el

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+":"; exec emacs --script "$0" -- "$@" # -*- mode: emacs-lisp; lexical-binding: t; -*-
```

Comment:
Nice hack---I'm wondering what the `":"` is needed for, though?

Reply:
Wonderful little monstrosity innit? That's how you do multiline shebangs with elisp.
https://rosettacode.org/wiki/Multiline_shebang#Emacs_Lisp